finance/proposal-calls/Moore_2021.md

```diff
@@ -0,0 +1,87 @@
+# Call for Astropy ideas / projects to be funded in 2021
+
+## Goals
+Projects and ideas funded through this process shall improve the Astropy Project. This includes code for the core repository, for coordinated or affiliated packages, infrastructure (e.g. testing, CI), tutorials, workshops, notebooks, community engagement, inclusion, diversity and empowerment, travel to conferences or meetings, talks, menotring, and anything else broadly Project related. This does not have to be a new initiative, but can be ongoing work (e.g. maintain the CI), but cannot include work that is already otherwise paid for (e.g. if you have a NASA grant to maintain the CI, you cannot bill for the same work twice).
+
+Suggested projects shall foster the Astropy Project goals of community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+Total funding for this call is coming from the Moore grant and 200,000 $ are available in total in the period spring 2021 to spring 2022.
+
+Project suggetions need to clearly identify what the funding is used for (e.g. "pay YYY $ to add Klingon Standard Time to astropy.time" or "organize astropy workshop to engage amateur astronomer community in Antarctica")
+
+*Note: You can think of this as an internal "proposal process" to allow the community to select exactly where the money that the Astropy Project has will be spend. To avoid confusion with the external proposal that gave the money to Astropy (here: Moore foundation), we use the term "project" or "project team" here.*
```

Comment:
```suggestion
*Note: You can think of this as an internal "proposal process" to allow the community to select exactly where the money that the Astropy Project has will be spend. To avoid confusion with the external proposal that gave the money to Astropy (here: Moore foundation), we use the term "suggested project" and "project team" here.*
```